.emacs.d/lisp/code/magit-c.el

```diff
@@ -1,5 +1,11 @@
-(setf magit-push-always-verify nil)
-(setf magit-last-seen-setup-instructions "1.4.0")
+(with-eval-after-load 'magit
+  (let ((popups-to-add-sign '(magit-merge-popup
+                              magit-rebase-popup)))
+    (mapcar (lambda (popup)
+              (magit-define-popup-option
+                popup
+                ?S "Sign using gpg" "--gpg-sign=" #'magit-read-gpg-secret-key))
+            popups-to-add-sign)))
```

Comment:
Where did `magit-push-always-verify` and `magit-last-seen-setup-instructions` go?  If they are not still needed, let me know.